src/exe/colmap.cc

```diff
@@ -1476,8 +1478,10 @@ int RunVocabTreeRetriever(int argc, char** argv) {
   options.AddDefaultOption("database_image_list_path",
                            &database_image_list_path);
   options.AddDefaultOption("query_image_list_path", &query_image_list_path);
-  options.AddDefaultOption("num_images", &num_images);
-  options.AddDefaultOption("num_verifications", &num_verifications);
```

Comment:
Why did you remove the ``num_verifications``? For large databases you probably only want to verify a shortlist of images?

Reply:
The existing behavior is to run a [fixed number of verifications]
 (https://github.com/colmap/colmap/blob/dev/src/retrieval/visual_index.h#L247), so it made sense to me to just specify num_images and add a boolean flag for whether to also run spatial verification. If we want to run spatial verification on only a subset of retrieved images and then return all retrieved images, then we could keep num_verifications.

Reply:
aha, somehow I thought the previous behavior was like that. Makes sense. Let’s keep your versio.